skaffold-plugins-core/build.gradle

```diff
@@ -83,7 +83,7 @@ check.dependsOn verifyGoogleJavaFormat
 
 /* CHECKSTYLE */
 checkstyle {
-    toolVersion = '8.10.1'
+    toolVersion = '8.11'
 
     // get the google_checks.xml file from the checkstyle jar and take out the java checks
     def googleChecks = resources.text.fromArchiveEntry(configurations.checkstyle[0], 'google_checks.xml').asString()
```

Comment:
at some point we should check if this is still hurting us, checkstyle java and google java format might have converged since doing this?

Reply:
Yep, filed #9